examples/README.md

```diff
@@ -0,0 +1,16 @@
+# Kubesu example
+
+1. Spin up a local cluster:
+   ```
+   kind create cluster
+   ```
+
+2. Apply example manifests
```

Comment:
maybe these two points as well (just suggestion)
- `kubectl create namespace default` (like used here https://github.com/Adhara-Tech/kubesu/pull/9/files?file-filters%5B%5D=.md&file-filters%5B%5D=.yml#diff-3ca6fa0450efd86b0719e8c49a79330fc15d296652b7e8923f2fa11224002a78R21)
- `kind load docker-image docker.io/adharatech/kubesu:v0.0.1-rc5`

Reply:
`default` namespace is created by default in kubernetes, it doesn't need to be created. The second point about loading the docker-image won't be needed once v0.0.1-rc5 is published, but it's true that it can be handy having these instructions while developing. If you agree, I will think on a nice way to document this and open another PR with that extended documentation (like adding a DEVELOPING.md doc or something 🤔 )